sceptre/scipool/config/strides/jumpcloud-idp.yaml

```diff
@@ -7,5 +7,7 @@ stack_tags:
 parameters:
   StridesAdminMetadata: !file_contents idp/jumpcloud/strides-admin.xml
   StridesCommunityMgrMetadata: !file_contents idp/jumpcloud/strides-communitymgr.xml
+  StridesViewernMetadata: !file_contents idp/jumpcloud/strides-viewer.xml
```

Comment:
This line has a typo @zaro0508 .. `StridesViewernMetdata`

Reply:
fixed